common/src/main/java/io/netty/util/LongBiConsumer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.util;
+
+/**
+ * Consumer of two {@code long} values.
+ */
+public interface LongBiConsumer {
```

Comment:
It was like that before, I made it generic based on an earlier comment. It is just like a [JDK 8 `BiConsumer`](https://docs.oracle.com/javase/8/docs/api/java/util/function/BiConsumer.html)